docs/eventing/samples/kafka/source/index.md

```diff
@@ -5,6 +5,8 @@ weight: 20
 type: "docs"
 ---
 
+# Apache Kafka Source Example
+
 Tutorial on how to build and deploy a `KafkaSource` [Eventing source](../../../sources/README.md) using a Knative Serving `Service`.
 
 ## Prerequisites
```

Comment:
Looks like numbering breaks using code blocks, have seen this in other places, we can take care of it during cleanup